spec/facets/AirflowRunFacet.json

```diff
@@ -0,0 +1,172 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/AirflowRunFacet.json",
+  "$defs": {
+    "AirflowRunFacet": {
```

Comment:
If this is an Airflow specific facet, why do we want to add it into global `spec`? 

When accepting such an exception, we can end up with dbt, spark, airflow, flink, snowflake, databricks, ... facets that blur the main goal of Openlineage which is: `whatever you do with data, OL event contains a technology agnostic description of that`.

Please consider replacing this with `RunParameterFacet` or some generic way to describe run details